core/vm/contracts.go

```diff
@@ -130,6 +130,8 @@ func init() {
 // ActivePrecompiles returns the precompiles enabled with the current configuration.
 func ActivePrecompiles(rules params.Rules) []common.Address {
 	switch {
+	case rules.IsFlareHardFork1:
```

Comment:
Two cases returning the same value. So they can be merged?

Reply:
No, they are evaluated in order.